src/cgame/cg_timerun_draw.c

```diff
@@ -856,7 +931,7 @@ void CG_DrawBannerPrint(void) {
 void CG_UpdateJumpSpeeds(void) {
 	int speed = 0;
 
-	if (cg.timerunJumpCounter < cg.predictedPlayerState.identifyClientHealth && cg.timerunJumpCounter < (int)(sizeof (cg.timerunJumpSpeeds) / sizeof (cg.timerunJumpSpeeds[0]))) {
+	if (cg.timerunJumpCounter < cg.predictedPlayerState.identifyClientHealth && cg.timerunJumpCounter < (int)(sizeof(cg.timerunJumpSpeeds) / sizeof(cg.timerunJumpSpeeds[0]))) {
```

Comment:
`sizeof` is a preprocessor directive not a C function. It should be followed by a space to highlight the difference with any C function.

Reply:
well done visual assist